src/ImageSharp/Formats/Gif/GifDecoderCore.cs

```diff
@@ -363,8 +380,6 @@ private unsafe void ReadFrameColors(byte[] indices, byte[] colorTable, int color
 
             if (this.previousFrame == null)
             {
-                this.metaData.Quality = colorTableLength / 3;
```

Comment:
Its not used anywhere (as I removed the usage of the metadata quality from the encoders).

The reason i removed using it tn the encoders was that it causes issues when switching formats, if you decoded a jpeg its 'quality' could be 70, but then encoding a gif would have caused a palette size of 70 which is drastically different. So now you have to be explicit but manually setting it on the encoders.